### ISSUE TITLE
Bigger widget

### ISSUE THREAD

**Issue by ghost** [davidtakac/prognoza#39]
Can you add bigger (4x4 for example) widget? I want it to be fully in center.

**Comment by davidtakac** [comment]
Hello! Do you mean just allow the current widget to expand vertically and center its content?

**Comment by ghost** [comment]
@davidtakac yes. But also add more info with 4x4 widget, for example, weather for the next day, kinda useless if it will be same 2x2 widget but with more black space.

**Comment by ghost** [comment]
Here is the screenshot: 
![widget](https://user-images.githubusercontent.com/110234273/210545706-057f4452-9091-4090-9a93-550fc242fd62.jpg)
I want to expand it, so it will take 50% of my home screen. But i can't, because it's only 2x2.
